src/doc/trpl/hello-cargo.md

```diff
@@ -1,58 +1,54 @@
-% Hello, Cargo!
+% Cargoの導入
 
-[Cargo][cratesio] is a tool that Rustaceans use to help manage their Rust
-projects. Cargo is currently in a pre-1.0 state, and so it is still a work in
-progress. However, it is already good enough to use for many Rust projects, and
-so it is assumed that Rust projects will use Cargo from the beginning.
+[Cargo][cratesio] はRustaceansがRustのプロジェクトを管理するために使用するツールです。
+
+Cargoはプレリリースの段階で、現在も開発中です。しかしすでに実用段階にあり、Rustのプロジェクトは当初からCargoを使用することを想定しています。
 
 [cratesio]: http://doc.crates.io
 
-Cargo manages three things: building your code, downloading the dependencies
-your code needs, and building those dependencies. At first, your
-program doesn’t have any dependencies, so we’ll only be using the first part of
-its functionality. Eventually, we’ll add more. Since we started off by using
-Cargo, it'll be easy to add later.
+Cargoはコードのビルド、依存関係の解決とビルドを行います。
+
+まずコードが依存関係を持たない場合、最初の機能のみ使用されます。
+やがてそのコードは依存関係を持つようになるでしょう。しかしCargoを使用するとそれが容易になります。
 
-If you installed Rust via the official installers you will also have Cargo. If
-you installed Rust some other way, you may want to [check the Cargo
-README][cargoreadme] for specific instructions about installing it.
+もしあなたがRustを公式のインストーラでインストールしたならば、Cargoも同時にインストールされています。
+もし他の方法でRustをインストールしたならば、[the Cargo README][cargoreadme] を参照してインストールしてください。
 
 [cargoreadme]: https://github.com/rust-lang/cargo#installing-cargo-from-nightlies
 
-## Converting to Cargo
 
-Let’s convert Hello World to Cargo.
+##Cargoの適用
+
+
+ではCargoをHello Worldに使用してみましょう。
+
+Cargoをプロジェクトに適用するためには、2つのことを行う必要があります。
+まず `Cargo.toml` という設定ファイルを作成し、次にソースファイルを適切な位置に保存する必要があります。
 
-To Cargo-ify our project, we need to do two things: Make a `Cargo.toml`
-configuration file, and put our source file in the right place. Let's
-do that part first:
+次のようにしてみましょう。
 
 ```bash
 $ mkdir src
 $ mv main.rs src/main.rs
 ```
 
-Note that since we're creating an executable, we used `main.rs`. If we
-want to make a library instead, we should use `lib.rs`.
-Custom file locations for the entry point can be specified
-with a [`[[lib]]` or `[[bin]]`][crates-custom] key in the TOML file described below.
+ここで注意すべき点は、実際に実行するソースファイルは名前を `main.rs` とし、ライブラリを使用する場合はライブラリのファイル名を `lib.rs`とする点です。もし他のディレクトリ名を使用したい場合は、[こちら(`[[lib]]` or `[[bin]]`)][crates-custom]を参照してTOMLファイルを設定してください。
```

Comment:
"create"という言葉が使われていますし、実際コンパイル言語ではないため、
> Note that since we're creating an executable, we used `main.rs`. If we want to make a library instead, we should use `lib.rs`.

を、
> 実行形式を作成する場合、ソースファイル名を`main.rs`とするよう留意して下さい。ライブラリを作成するのであれば`lib.rs`として下さい。

という感じにしたほうが良いのかなと思いました。
なお[本家RustBook](https://doc.rust-lang.org/book/hello-cargo.html)が更新されていました。
この行と続きの行の間に、
> This convention is required for Cargo to successfully compile our projects, but it can be overridden if we wish. 

が続きます。追加お願いします。